### ISSUE TITLE
Are there any versions of Mypal newer than version 68.12.3beta? Is it possible to download them then and where? Suggest a link.

### ISSUE THREAD

**Issue by kiza124** [Feodor2/Mypal#104]
I look at the development of the Mypal browser ended with version 68.12.3beta and there are still no new versions. And this version somehow distortedly opens individual sites, for example, https://aliexpress.ru/item/1005004622026710.html?spm=a2g0o.search.search_results.5.d90b26d4Yiiifd&sku_id=12000029865879264
- elements of the page seem to run into one another. Although I don’t know, maybe it’s impossible to make a browser that works in Win Xp and at the same time opens all sites correctly - the way the creator of the site intended ?!
Or is there a newer version? Is it possible to download them then and where? Tell me the link, because. http://www.mypal-browser.org/release/ contains 68.12.3beta and older.

**Comment by IP-CAM** [comment]
Here i the latest Release:
https://github.com/Feodor2/Mypal68

**Comment by kiza124** [comment]
I have previously visited this section - https://github.com/Feodor2/Mypal68, but did not understand (did not find) how you can download a newer version there.

**Comment by IP-CAM** [comment]
https://github.com/IP-CAM/Mypal68-Windows-XP-Browser

**Comment by kiza124** [comment]
Where should I click to download the browser?

**Comment by IP-CAM** [comment]
You don't seem to have much Web-Knowledge, it's in the 
CODE / DOWNLOAD ZIP
Section.

**Comment by kiza124** [comment]
The link https://github.com/IP-CAM/Mypal68-Windows-XP-Browser CODE / DOWNLOAD ZIP contains a 627 MB archive! This is clearly not what I need. Mypal version 68.12.3beta now takes 170 MB on my disk, so the 627 MB version is clearly something that I don’t need. Or do you offer me to download some version that needs to be compiled somehow or do something else with it to make it work?! Few people need this option either, unless some programmers

**Comment by IP-CAM** [comment]
Well, it's your decision to use it or not, you can also download it from the official Site, to get the same, if you believe to get a modified Download from my Site:
https://github.com/Feodor2/Mypal68
Good Luck.

**Comment by kiza124** [comment]
Why does Mypal version 68.12.3beta take 170 MB and the newer version takes 627 MB? What did you do there to make him take so much?! No, if it's that size, I don't need it.

**Comment by kiza124** [comment]
You have version 68.12.3beta on your site http://www.mypal-browser.org/release/ - a 60 MB archive. Can you make the 68.12.5 version the same so that it takes about the same size and put it at the same address http://www.mypal-browser.org/release?

**Comment by Methen** [comment]
The dowload does not appear to even be working

**Comment by kiza124** [comment]
Well, in general, can you make the version of mypal 68.12.5 in the same form so that it takes about the same size (60 mb) and put it somewhere where you can download it and use it later to work on a Windows XP computer ?

**Comment by Methen** [comment]
Where do I download the latest version at the one link on githum was not working right ?
  ----- Original Message -----

**Comment by jonm58** [comment]
https://github.com/Feodor2/Mypal68/

**Comment by Methen** [comment]
Not certain what you sent me but it was some hugh file and does not ever appaer to have a short cut to open the browser I just wanted the latest version of the browser ...
  ----- Original Message -----

**Comment by jonm58** [comment]
https://github.com/Feodor2/Mypal68/releases/download/68.12.5b/mypal-68.12.5.en-US.win32.zip

**Comment by Methen** [comment]
Thank you, that worked, A little confused about something I have the old version of the folder in my documents, when I unzipped the one you sent me and opened up it already have my nook marks in it that  does not male much sense. sense on folder is in my ducements folder and the other is on the desp top...
  ----- Original Message -----

**Comment by kiza124** [comment]
Thank you. But, and this version somehow distortedly opens individual sites, for example, https://aliexpress.ru/item/1005004622026710.html?spm=a2g0o.search.search_results.5.d90b26d4Yiiifd&sku_id=12000029865879264
- elements of the page seem to run into one another. Although I don’t know, maybe it’s impossible to make a browser that works in Win Xp and at the same time opens all sites correctly - the way the creator of the site intended ?!

**Comment by Methen** [comment]
Yes but  htis version still has the problem of not being able to add programs in hte applacations list in the tools options cant get utorrrent to be added and when I try to open it on pirate bay I get a browser does not know how to handle it option
  ----- Original Message -----

**Comment by Martin-Armsby** [comment]
Hi,


I'm setting up an old favorite laptop - its a Dell Latitude D400 which I used for audio Studio productions, multitrack recordings etc.
I was pleased to find this browsing effort but it is not working for youtube.
Message says there is a script not responding. 
So I tried the 86 versions in zipped format.
They dont work at all and also have no installer.
Has the project been abandoned ?

**Comment by Templayer** [comment]
I think youtube works with the newer New Moon and Serpent browsers. You could also try other Goanna-based browsers, such as Centaury, IceApe-UXP and BNavigator.

Roytam frequently releases new versions, he's a WinXP god. But I liked MyPal too. :<

http://rtfreesoft.blogspot.com/

The links with Basilisk in their name are for the Serpent browser. Pale Moon for XP is called New Moon.
Basilisk (and thus Serpent) is the more experimental version of Pale Moon (New Moon).

Also, there is the more modern version of K-meleon (the Goanna engine based version) for XP there.

NM28XP is short for New Moon 28 XP.

I would recommend trying out the New Moon browser. I remember a few years back even trying out Twitch it in and it worked.

Win64 version if you are using a 64-bit version of XP,
Win32 SSE if your CPU supports SSE,
Win32 IA32 if you have an i386 CPU,
or the plain Win32 if you have none of those.

It's too bad that MyPal seems abandoned. :(

**Comment by Methen** [comment]
I do not have a problem with youtube just sites like face book  messenger and sites telling me the browser needs to be updated



----- Original Message -----

**Comment by Templayer** [comment]
I was replying to the previous comment. The world doesn't turn around you, you know? This one:

> I'm setting up an old favorite laptop - its a Dell Latitude D400 which I used for audio Studio productions, multitrack recordings etc.
> I was pleased to find this browsing effort but it is not working for youtube.

Facebook should work too. Worst case scenario - you would have to spoof the Navigator.userAgent to make Facebook think that you are using a modern browser and see if that works.

**Comment by Methen** [comment]
I was just responding to the email I got 
  ----- Original Message -----

**Comment by Templayer** [comment]
Because you probably have it set to be notified of any comment on this Issue.

That means that if anybody comments on this Issue, you will be sent an email about it. It doesn't mean they are replying to you directly.

**Comment by Methen** [comment]
I see...
  ----- Original Message -----

**Comment by jonm58** [comment]
https://github.com/Feodor2/Mypal68/issues/160#issuecomment-1442575016
Mypal 68.13
